app/main.py

```diff
@@ -1,3 +1,51 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers"
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        if not isinstance(other, Distance):
+            return Distance(self.km + other)
+        else:
+            return Distance(self.km + other.km)
+
+    def __iadd__(self, other):
+        if not isinstance(other, Distance):
+            self.km += other
+            return self
+        else:
+            self.km += other.km
+            return self
+
+    def __mul__(self, other):
+        return Distance(self.km * other)
+
+    def __truediv__(self, other):
+        if not isinstance(other, Distance):
+            return Distance(round(self.km / other, 2))
+        else:
+            return Distance(round(self.km / other.km, 2))
+
+    def __lt__(self, other):
+        return self.km < other
+
+    def __gt__(self, other):
+        return self.km > other
+
+    def __eq__(self, other):
+        return self.km == other
+
+    def __le__(self, other):
+        return self.km <= other
+
+    def __ge__(self, other):
+        return self.km >= other
```

Comment:
other can be of type Distance, then your code will not work. Consider this in the following methods